packages/repo-tools/src/commands/api-reports/api-extractor.ts

```diff
@@ -213,10 +211,19 @@ ApiReportGenerator.generateReviewFileContent =
       collector,
       ...moreArgs,
     );
-    return prettier.format(content, {
-      ...require('@spotify/prettier-config'),
-      parser: 'markdown',
-    });
+
+    try {
+      const prettier = require('prettier') as typeof prettierType;
```

Comment:
all this typing maybe os more confusing than helping here but was the only way i find to have proper `prettier` typings.
Should test the behaviour without prettier installed, and if it casu any problem will remove all the type import

Reply:
tested and seems to work fine without any `prettier` installed, so keeping it

Reply:
You can `typeof import('prettier')` as well, but this works too :grin:

Reply:
ohh didn’t think about that, thanks!!